src/richie/apps/search/filter_definitions/courses.py

```diff
@@ -26,6 +29,26 @@ class IndexableFilterDefinition(TermsAggsMixin, TermsQueryMixin, BaseFilterDefin
     number of choices.
     """
 
+    def __init__(self, name, reverse_id=None, **kwargs):
+        self.reverse_id = reverse_id
+        super().__init__(name, **kwargs)
+
+    @cached_property
+    def aggs_include(self):
```

Comment:
Missing a docstring

Reply:
I still don't understand why pylint ignores this file :thinking: